arch/arm/boot/dts/broadcom/bcm2712d0-rpi-5-b.dts

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
This whole commit could be dropped now that we have the `bcm2712d0` overlay, unless we're catering for upstream preferences.

Reply:
Yeah, I was thinking it will be wanted in future for upstream.